projects/core/src/features-config/feature-toggles/config/feature-toggles.ts

```diff
@@ -43,6 +43,11 @@ export interface FeatureTogglesInterface {
    */
   productConfiguratorAttributeTypesV2?: boolean;
 
+  /**
+   * Enable strict error handling in SSR for errors occurred in HTTP calls and in NgRx failure actions.
+   */
+  strictHttpAndNgrxErrorHandling?: boolean;
```

Comment:
I had the same considerations yesterday and I've started wondering what if we eventually enable this feature for CSR?

I see two possible scenarios:
* introduce strict error handling for CSR when the toggle for SSR error handling is still valid/active (12 moths rule)
* introduce strict error handling for CSR when SSR have it by default (after 12 months)

I think both scenarios lead to introducing separate toggle `csrErrorHandling`, `strictNgrxAndHttpErrorHandlingInCsr`, you name it, so I would say we can go with renaming this existing toggle to be more related to SSR. I'm still not sure whether `ssrErrorHandling` is ok as in this case, it might be too generic and not describe what exactly we're toggling. But maybe good JSDoc (proposed by you in the comment above) is good enough 🤔